pkg/api/v1alpha1/cluster_types.go

```diff
@@ -51,7 +51,8 @@ type ClusterSpec struct {
 	// Sysctl properties to be applied during initialization
 	// given as a list of key=value pairs.
 	// Example: fs.aio-max-nr=232323
-	Sysctls []string `json:"sysctls,omitempty"`
+	Sysctls    []string `json:"sysctls,omitempty"`
+	ScyllaArgs string   `json:"scyllaArgs,omitempty"`
```

Comment:
now it's required, but should be optional

Reply:
scylla.scylladb.com_clusters.yaml resulted in 
```
          required:
          - agentVersion
          - datacenter
          - version

```
So it is not required on scheme level, also I've checked it by deploying cluster with no scyllaArgs and it passed thru